_posts/2020-07-15-#19031.md

```diff
@@ -12,17 +12,22 @@ commit:
 
 ## Notes
 
-* [PR 19031](https://github.com/bitcoin/bitcoin/pull/19031) is the
-  implementation of [BIP
-  155](https://github.com/bitcoin/bips/blob/master/bip-0155.mediawiki), the
-  `addrv2` message.
+* [PR 19031](https://github.com/bitcoin/bitcoin/pull/19031) is a proposed implementation of the [BIP 155](https://github.com/bitcoin/bips/blob/9286b5254317d9e73fb25c5f0acd2b2d9937843e/bip-0155.mediawiki) `addrv2` messages, a new p2p message format proposed in early 2019 by Wladimir J. van der Laan to gossip longer node addresses. This is required to support new-generation Onion addresses, I2P, and potentially other networks that have longer endpoint addresses than fit in the 128 bits of the current `addr` message.
```

Comment:
One last thing (sorry): s/new/next/

Reply:
(What *is* I2P?)

Reply:
(If it isn't obvious to everyone but me, maybe write out what it stands for)

Reply:
Google tells me it stands for "Invisible Internet Project" (It's one of those _smart_ initialisms).  I suspect it's one of those things that people know by the abbreviation and not the full word, let's leave it :)